src/e3/fs.py

```diff
@@ -225,8 +235,15 @@ def compute_state(file_path: str) -> bytes:
                 full_path = os.path.join(root, path)
                 result.update(compute_state(full_path))
 
+                if hash_content:
```

Comment:
The `if hash_content:` block should be placed outside if/else, since it is common to both if and else.

Reply:
Nope ;) Here the result is updated in a loop, for every file :)

Reply:
Yes, it's true, I hadn't seen that!